db/migrate/20180116163006_store_signon_user_ids.rb

```diff
@@ -0,0 +1,9 @@
+class StoreSignonUserIds < ActiveRecord::Migration[5.1]
+  def change
+    add_column :content_changes,    :signon_user_uid, :string
```

Comment:
I wonder if we should take the opportunity now to clear these tables and add this column in as `null: true`?

Reply:
Sorry @thomasleese I missed that you commented days ago. I'm assuming typo'd `null: false`also.

Seems a reasonable step to require it to not be null. I feel like adding the clearing out of tables into this PR overloads things a bit though. Are we going to be planning to do a wipe pre-launch? we could write a story for that nowish and flag setting these as not null to it.

Reply:
Yeah, okay, if we add a card to do that later then that's fine.